src/status_im/utils/keychain/core.cljs

```diff
@@ -77,3 +143,25 @@
 
 (defn set-username []
   (when platform/desktop? (.setUsername rn/keychain username)))
+
+;;;; Effects
+
+(re-frame/reg-fx
+ :save-user-password
+ (fn [[address password]]
+   (save-user-password address password
+                       #(if (not %)
```

Comment:
the same comment regarding `when-not`, also if you put `address ` param to the next line it will make following lines shorter, i hope we still want to have <= 80 chars in line